src/common/view-definitions/version-4/split-definition.ts

```diff
@@ -95,21 +96,21 @@ function fromSort(sort: Sort): SplitSortDefinition {
   return { ref, ...rest };
 }
 
-function toLimit(limit: unknown): number | null {
+function toLimit(limit: unknown, dimension: Dimension): number | null {
   if (limit === null) return null;
   if (isNumber(limit) && isFiniteNumber(limit)) return limit;
-  return AVAILABLE_LIMITS[0];
+  return dimension.limits[0];
```

Comment:
encapsulate into dim.defaultLimit

Reply:
I can extract this with better name, but this is very specific behaviour for split definition converter. 
It is not "dimensions default limit value" but more like "I got malformed data and need to pick something".

I will keep an eye on this when implementing #756, where visualisation picks "default limit value" for split.